### PR TITLE
feat: add product model

### LINKED ISSUE #1: Criar modelo de Produto

### PR DESCRIPTION
**Pull request by maranbrasil**
- Create a model for product class.

close #1

### CODE REVIEW

**Review by evandrosutil — APPROVED** [review]

**Review by Paulorlima — APPROVED** [review]

**Review by pedroLSoares — APPROVED** [review]
LGTM!

**Review by Icaro-Salgado — APPROVED** [review]
Muito bom!

**Review by MeliKlin — APPROVED** [review]
It seems good to me.

**Review by ThainanEsteves — APPROVED** [review]

File: src/main/java/br/com/mercadolivre/desafiospring/models/Product.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Product {
+    
+    private Long productId;
+    private String name;
+    private	String category;
```
Comment (evandrosutil) [inline src/main/java/br/com/mercadolivre/desafiospring/models/Product.java:16]:
```suggestion
    private	String category;
```

Comment (Icaro-Salgado) [reply src/main/java/br/com/mercadolivre/desafiospring/models/Product.java:16]: To confuso

**Pull request merged**